webviews/store.js

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const filters = writable(['Repository']);
```

Comment:
You're declaring a `filters` here as a writable, I don't see `get(filters)` anywhere?
Or does this line: https://github.com/Open-Sourced-Olaf/vscode-github-projects/blob/d5e13573e8ce756f8d52e778b1efa6b8555dffba/webviews/components/ProjectsList.svelte#L12

automatically gets the value from svelte store?

Reply:
I was playing with the store but couldn't get it working, let me remove that

Reply:
@YuanSamuel  ikr, Svelte store felt like a mess with us wanting svelte to re-render when there's a change in the store. :/

Reply:
He might have first tried cross-component store.js update and subscription, and instead he is sending `filters` as props from Home.svelte.  Can anyone confirm that store.js is not actually doing anything?  See?  @aitikgupta is truly a genius in questioning this assumption.